vowpalwabbit/offset_tree.cc

```diff
@@ -138,13 +142,22 @@ namespace VW { namespace offset_tree {
       return scores;
     }
 
+    // Save and clear example label.  
+    // (CB algs predict_or_learn checks for valid label during learn.)
+    static thread_local CB::label saved_label;
```

Comment:
This shouldn't (conceptually) be necessary since label should not affect prediction.  Can we instead fix whatever the issue is in cb_algs?